content/en/docs/components/pipelines/user-guides/core-functions/connect-api.md

```diff
@@ -142,177 +136,246 @@ subjects:
 for a list of some important `pipelines.kubeflow.org` RBAC verbs.
 * Kubeflow Notebooks pods run as the `default-editor` ServiceAccount by default, so the RoleBindings for `default-editor` apply to them
 and give them access to submit pipelines in their own namespace.
+* For more information about profiles, see the [Manage Profile Contributors](/docs/components/central-dash/profiles/#manage-profile-contributors) guide.
 {{% /alert %}}
 
 </details>
 
-## Full Kubeflow <sub>(from outside cluster)</sub>
+### **Kubeflow Platform - Outside the Cluster**
 
 <details>
 <summary>Click to expand</summary>
 <hr>
 
-The process to authenticate the Pipelines SDK from outside the cluster in multi-user mode will vary by distribution:
-
-* [Kubeflow on Google Cloud](/docs/distributions/gke/pipelines/authentication-sdk/#connecting-to-kubeflow-pipelines-in-a-full-kubeflow-deployment)
-* [Kubeflow on AWS](/docs/distributions/aws/pipeline/#authenticate-kubeflow-pipeline-using-sdk-outside-cluster)
-* [Kubeflow on Azure](https://awslabs.github.io/kubeflow-manifests/docs/component-guides/pipelines/)
-* [Kubeflow on IBM Cloud](/docs/distributions/ibm/pipelines/#2-authenticating-multi-user-kubeflow-pipelines-with-the-sdk)
-
-### Example for Dex
+{{% alert title="Tip" color="info" %}}
+The process to authenticate the Pipelines SDK from outside the cluster will vary by [Kubeflow Distribution](/docs/started/installing-kubeflow/#packaged-distributions) and identity provider.
```

Comment:
I think this DEX part should link to the Kubeflow Platform page in general https://www.kubeflow.org/docs/started/installing-kubeflow/#kubeflow-platform instead of directly to the distributions. Furthermore we should say that below DEX there is oauth2-proxy that supports machine to machine authentication with Kubernetes Tokens.

Reply:
Updated link in https://github.com/kubeflow/website/pull/3795/commits/95d20ea79e41634a4c7f003452eb5497ee86b0fc.

Let's raise a separate PR which adds a "Kubeflow Platform - Dex Token Exchange", or whatever you want to call it, presuming you are talking about the ability of Dex to exchange external JWTs for its own ones:
- https://dexidp.io/docs/guides/token-exchange/